react/react2/week1/homework/src/App.js

```diff
@@ -23,28 +26,31 @@ function App() {
   useEffect(() => {
     fetchAPI()
   }, [fetchAPI])
+  const title = "Github user searcher"
+  const AppValues = [title]
   return (
     <div className="App">
-      <h1> Github user searcher </h1>
+      {/* All state should be managed using the context api! */}
+      <AppContext.Provider value={AppValues}>
```

Comment:
context is not needed here, because all your components are on the same "level", and there is not so much state